src/util/spectra.jl

```diff
@@ -55,20 +55,23 @@ function apply_doppler_boost!(spectra::AS, dict::AbstractDict ) where { AS<:Abst
         doppler_factor *= calc_doppler_factor(dict[:drift_rv])
     end
     =#
-    if !haskey(dict,:ssb_rv) && !haskey(dict,:ssbz) && !have_issued_ssb_warning
+    if !haskey(dict,:ssb_rv) && !haskey(dict,:ssbz) && !haskey(dict,:ssb_rv_kmps) && !have_issued_ssb_warning
          @info "apply_doppler_boost! didn't find :ssb_rv or :ssbz to apply."
          have_issued_ssb_warning = true
     end
     if  haskey(dict,:ssbz)
         doppler_factor   *= calc_doppler_factor(z=dict[:ssbz])
     elseif  haskey(dict,:ssb_rv)
         doppler_factor   *= calc_doppler_factor(dict[:ssb_rv])
+    elseif  haskey(dict,:ssb_rv_kmps)
+        doppler_factor   *= calc_doppler_factor(rv_kmps=dict[:ssb_rv_kmps])
     end
     if !haskey(dict,:diff_ext_rv ) && !have_issued_diffext_warning
         @info "apply_doppler_boost! didn't find :diff_ext_rv to apply."
         have_issued_diffext_warning = true
     end
     if  haskey(dict,:diff_ext_rv)  doppler_factor   *= calc_doppler_factor.(dict[:diff_ext_rv])  end
+    if  haskey(dict,:binaryRV)  doppler_factor   *= calc_doppler_factor.(dict[:binaryRV])  end
```

Comment:
When there is a binary, should we be applying the correction to the wavelength scale before calculating things?  Or should we be measuring an RV and subtracting off the binary RV at the end?

Reply:
decided to go with this way in case there is a large enough binary RV that it is important to do the analysis in the frame of the star instead of the center of mass of the binary frame.